selfdrive/ui/paint.cc

```diff
@@ -70,10 +71,12 @@ static void draw_chevron(UIState *s, float x_in, float y_in, float sz,
                           NVGcolor fillColor, NVGcolor glowColor) {
   float x, y;
   if (!car_space_to_full_frame(s, x_in, y_in, 0.0, &x, &y)) {
-    return;
+    if (x < 0 || y < 0) 
+      return;
   }
 
   sz = std::clamp((sz * 30) / (x_in / 3 + 30), 15.0f, 30.0f);
+  y = std::min(s->scene.viz_rect.bottom() - sz / 2,  y);
```

Comment:
since y is already a screen coordinate, we can make sure it's always displayed here